README.md

```diff
@@ -35,6 +35,10 @@ Obtain the publishable and secret keys from the developer test mode on the Strip
 
 * Open the cloud-config.yaml file and replace the hard-coded name with your own.
 
+* Change the path on line 75 in vm_deploy.sh to /home/yourname
+
+* If preferred, change line 82 in vm_deploy.sh to yourname-docker
```

Comment:
Recommend to use $USER instead of yourname or username so you get `$USER-docker` and it will dynamically put in the current user's name.

Reply:
Changed this in the recent Firebase PR: crown-apparel-container for container name and crown-apparel-image.

Will change README.md to either after Firebase review.